recipes/FastTree/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: fasttree
+  version: "2.1.8"
+
+source:
+  fn: FastTree-2.1.8.c
+  url: http://www.microbesonline.org/fasttree/FastTree-2.1.8.c
+
+build:
+  number: 0
+  script:
+    - mkdir -p "$PREFIX/bin"                                  # [osx or linux]
+
+    - gcc -DUSE_DOUBLE -O3 -finline-functions -funroll-loops -Wall -o FastTree FastTree-2.1.8.c -lm
+    - gcc -fopenmp -DUSE_DOUBLE -O3 -finline-functions -funroll-loops -Wall -o FastTreeMP FastTree-2.1.8.c -lm
+
+    - mv -v ./FastTree* $PREFIX/bin                           # [osx or linux]
+    - xcopy /s "FastTree.exe" "%PREFIX%\bin"                  # [win64]
+
+test:
+  commands:
+    - FastTree -help                                          # [osx or linux]
+    - FastTreeMP -help                                        # [osx or linux]
```

Comment:
A real test here would be good. Especially a test for OpenMP and `export OMP_NUM_THREADS=2`.